### PR TITLE
Cleaning formatting for Replace Spark Cluster Manager chapter

### PR DESCRIPTION
**Pull request by ph07**

### CODE REVIEW

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md [hunk 1]
```diff
@@ -164,7 +166,7 @@ You can clean up the index.html file when you are done by running: `sudo rm inde
 
 Make sure the README.md is in pwd. Then enter:
 
-```
+```Spark-Shell
```
Comment (TheOnlyRew) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Why `Spark-Shell`? This looks a lot like JavaScript to me, should we use that highlighting?

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md [hunk 2]
```diff
@@ -40,18 +40,18 @@ To replace your Spark Cluster Manager with the BDP cluster manager, you will do
 
 1. Add BDP service configs for spark-master and spark-worker.
 2. Activate the services.
-3. Verify
+3. Verify your success.
```
Comment (TheOnlyRew) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: This language is somewhat strange. Perhaps "Verify the services are running"?

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md [hunk 3]
```diff
@@ -27,11 +27,11 @@ Before you replace Spark Standalone Cluster Manager with the one provided by BDP
...
 * Have the following information available:
-IP addresses of Riak cluster nodes and protobuf ports. They will look like: 172.31.9.125:10017,172.31.9.126:10017,172.31.9.127:10017.
-IP addresses and ports of LES service. They will look  like: 172.31.9.125:10012,172.31.9.126,172.31.9.127:10012.
-Name of the group for leader election. (It is set to 'spark_leaders' by default.)
-Name of consistent bucket for Spark metadata storage (you set this up during [configuration][bdp configure]).
-Name of CRDT map for metadata storage (i.e. 'spark-cluster-map'). The CRDT map stores serialized classes containing Workers, Applications and Jobs data. This data is stored in the map as a key-value pair with key composed of a prefix of entity type plus unique id (i.e. app_42343sdfd33), and value containing serialized (by serializer passed in by spark master) class for Worker, Application or Job.
+  * IP addresses of Riak cluster nodes and protobuf ports. They will look like: `172.31.9.125:10017, 172.31.9.126:10017, 172.31.9.127:10017`.
+  * IP addresses and ports of LES service. They will look  like: `172.31.9.125:10012, 172.31.9.126:10012, 172.31.9.127:10012`.
```
Comment (TheOnlyRew) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Should these (and the above line) be:

> They will look  like: `172.31.9.125:10012`, `172.31.9.126:10012`, `172.31.9.127:10012`.

or do we want the whole list in a single code element?

Comment (ph07) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Either way, imho

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md (hunk 2 above)
Comment (ph07) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Yes, your version is better. Need to change in both places (here and below)

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md (hunk 1 above)
Comment (ph07) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Afaik, this is Scala code, which is entered into Spark Shell. We can use 'Scala' for proper syntax highlight, but I wanted to sow it is done inside Spark Shell, not shell or IDE.

**Comment by ph07** [comment]
@paegun - should this whole chapter marked as "Enterprise only"?

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md (hunk 2 above)
Comment (laurenrother) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Here as well, @ph07 , would you like to update this PR, or should I make the change myself?

**Comment by paegun** [comment]
Yes @ph07 anything to do with Redis or Cache Proxy or Leader Election Service are Enterprise only.

File: source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md (hunk 3 above)
Comment (laurenrother) [inline source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md]: Neither. They should not be in code elements. This is simply a list of information a user needs, not specific commands to obtain it or quotes from its location within a codeblock.

**Comment by laurenrother** [comment]
@ph07 At this point, this page is a mishmash of edits in-flight with James' work and serious additions to the page. I'm going to keep this PR open for awhile, as I make these changes and updates myself.

**Comment by laurenrother** [comment]
https://github.com/basho/basho_docs/commit/111c1961728f50722ff0a191d936d41ea02c7912

**Pull request closed without merging**